lib/ui/src/modules/ui/components/layout/index.test.js

```diff
@@ -17,15 +19,12 @@ describe('manager.ui.components.layout.index', () => {
         />
       );
 
-      const markup = wrap.html();
-      expect(markup).toMatch(/LeftPanel/);
-      expect(markup).toMatch(/DownPanel/);
-      expect(markup).toMatch(/Preview/);
+      expect(wrap).toMatchSnapshot();
```

Comment:
This snapshot is huge, is there some sane way to deal with that, or is it inevitable?

Reply:
We can explicitly test the presence of particular styles instead

Reply:
Did that